docs/source/model_doc/visual_bert.rst

```diff
@@ -58,7 +58,9 @@ layer, and is expected to be bound by [CLS] and a [SEP] tokens, as in BERT. The
 appropriately for the textual and visual parts.
 
 The :class:`~transformers.BertTokenizer` is used to encode the text. A custom detector/feature extractor must be used
-to get the visual embeddings. For an example on how to generate visual embeddings, see the `colab notebook
+to get the visual embeddings. Refer to the `demo notebook
+<https://github.com/huggingface/transformers/tree/master/examples/research_projects/visual_bert>`__ for an example on
+VisualBERT VQA. For an example on how to generate visual embeddings, see the `colab notebook
```

Comment:
(nit) Could you maybe format these examples in a list?

Reply:
Changed ^_^